src/core/authenticationKey.ts

```diff
@@ -48,9 +48,14 @@ export class AuthenticationKey {
   }
 
   /**
-   * Creates an AuthenticationKey from seed bytes and a scheme
+   * Derives an AuthenticationKey from the public key seed bytes and an explicit derivation scheme.
+   *
+   * This facilitates explicitly targeting a specific derivation scheme for an authentication key
+   * to use for an account.
+   *
+   * This is useful for deriving an account address from an account whose address
+   * was derived using a legacy derivation scheme.
```

Comment:
it is not only for "legacy derivation scheme", right?

Reply:
Yea, I probably should have said `This is particularly useful for...` but even then it's a little misleading.

I think it should probably just be the first two sentences,  I'll change it to that